docker-compose.README.md

```diff
@@ -29,11 +29,11 @@ can be verified correctly.
 
 The following users and permissions are available:
 
-| User        | Password    | Office | Permissions    |
-| ----------- | ----------- | ------ | ------------   |
-| l2hectest   | l2hectest   | SPK    | General User   |
-| l1hectest   | l1hectest   | SPL    | No permissions |
-| m5hectest   | m5hectest   | SWT    | General User   |
+| User                  | Password    | Office | Permissions    |
+| --------------------- | ----------- | ------ | ------------   |
+| l2hectest.1234567890  | l2hectest   | SPK    | General User   |
```

Comment:
Is username.EDIPI convention in the CWMS database?    It feels like an attribute being attached to the username.

Reply:
No that's a quirk of keycloak and how it's using the CAC "CN=" portion of the user certificate. That said if it's confusing I can see if I can set it to the normal name but still have the JWT provide the version with the edipi.